modular_skyrat/modules/food_replicator/replicator_designs/replicator_medical.dm

```diff
@@ -0,0 +1,67 @@
+/datum/design/sutures
+	name = "sutures"
+	id = "slavic_suture"
+	build_type = BIOGENERATOR
+	materials = list(/datum/material/biomass = 150)
+	build_path = /obj/item/stack/medical/suture
+	category = list(RND_CATEGORY_INITIAL, RND_CATEGORY_NRI_MEDICAL)
+
+/datum/design/mesh
+	name = "regenerative meshes"
+	id = "slavic_mesh"
+	build_type = BIOGENERATOR
+	materials = list(/datum/material/biomass = 150)
+	build_path = /obj/item/stack/medical/mesh
+	category = list(RND_CATEGORY_INITIAL, RND_CATEGORY_NRI_MEDICAL)
+
+/datum/design/bruise_patch
+	name = "bruise patch"
+	id = "slavic_bruise"
+	build_type = BIOGENERATOR
+	materials = list(/datum/material/biomass = 250)
+	build_path = /obj/item/reagent_containers/pill/patch/libital
+	category = list(RND_CATEGORY_INITIAL, RND_CATEGORY_NRI_MEDICAL)
+
+/datum/design/burn_patch
+	name = "burn patch"
+	id = "slavic_burn"
+	build_type = BIOGENERATOR
+	materials = list(/datum/material/biomass = 250)
+	build_path = /obj/item/reagent_containers/pill/patch/aiuri
+	category = list(RND_CATEGORY_INITIAL, RND_CATEGORY_NRI_MEDICAL)
+
+/datum/design/gauze
+	name = "medical gauzes"
+	id = "slavic_gauze"
+	build_type = BIOGENERATOR
+	materials = list(/datum/material/biomass = 100)
+	build_path = /obj/item/stack/medical/gauze
+	category = list(RND_CATEGORY_INITIAL, RND_CATEGORY_NRI_MEDICAL)
+
+/datum/design/epi_pill
+	name = "epinephrine pill"
+	id = "slavic_epi"
+	build_type = BIOGENERATOR
+	materials = list(/datum/material/biomass = 75)
+	build_path = /obj/item/reagent_containers/pill/epinephrine
+	category = list(RND_CATEGORY_INITIAL, RND_CATEGORY_NRI_MEDICAL)
+
+/datum/design/salb_pill
+	name = "salbutamol pill"
+	id = "slavic_salb"
+	build_type = BIOGENERATOR
+	materials = list(/datum/material/biomass = 75)
+	build_path = /obj/item/reagent_containers/pill/salbutamol
+	category = list(RND_CATEGORY_INITIAL, RND_CATEGORY_NRI_MEDICAL)
```

Comment:
Salbutamol is incredibly good, I really don't think this is a thing that should be in the NRI food replicator.

Reply:
I wish there were other suffocation-healing chems but oh well will remove it ig. :(

Reply:
Will replace with tirimol pills then ig.